common-information/mvvm-helpers/common-converters.md

```diff
@@ -49,31 +50,48 @@ This converter converts __Boolean__ values to 1 or 0 opacity. If the __Boolean__
 
 ## BooleanToVisibilityConverter
 
-This converter converts __Boolean__ values to __Visibility__ enumeration values. If the __Boolean__ value is __false__ the __Convert()__ method will return __Visibility.Collapsed__.
+This converter converts __Boolean__ values to __Visibility__ enumeration values. If the __Boolean__ value is __false__ the `Convert()` method will return __Visibility.Collapsed__.
 
 ## InvertedBooleanToVisibilityConverter
 
-This converter converts __Boolean__ values to __Visibility__ enumeration values. If the Boolean value is __true__ the __Convert()__ method will return __Visibility.Collapsed__.
+This converter converts __Boolean__ values to __Visibility__ enumeration values. If the Boolean value is __true__ the `Convert()` method will return __Visibility.Collapsed__.
 
 ## VisibilityToBooleanConverter
 
-This converter converts values from the __Visibility__ enumeration to __Boolean__ values. The __Convert()__ method returns __true__ if the passed value is __Visibility.Visible__ and __false__ otherwise.
+This converter converts values from the __Visibility__ enumeration to __Boolean__ values. The `Convert()` method returns __true__ if the passed value is __Visibility.Visible__ and __false__ otherwise.
 
-To invert the results of the converter, set the __IsInverted__ property. This way, __Visibility.Visible__ will be converted to __false__ and __Visibility.Collapsed__ to __true__.
+To invert the results of the converter, set the `IsInverted` property. This way, __Visibility.Visible__ will be converted to __false__ and __Visibility.Collapsed__ to __true__.
 
-#### __[C#] Example 2: Setting the view model__
-{{region xaml-common-converters_1}}
+#### __[C#] Setting the IsInverted property__
+{{region common-converters-1}}
 	<FrameworkElement.Resources>
 		<telerik:VisibilityToBooleanConverter x:Key="InvertedVisibilityToBooleanConverter" IsInverted="True"/>
 	</FrameworkElement.Resources>
 {{endregion}}
 
+## EnumToBooleanConverter
+
+The `EnumToBooleanConverter` converts an __Enum__ to a __Boolean__ value. The `Convert()` method return __true__ if the provided value matches the __Enum__ value that is provided in the `CommandParameter` property. If the values do not match, the method will return __false__.
```

Comment:
Explain that the ConvertParameter can contain multiple Visibility values. I think the separator was comma, but check this again. You can add a second example showing how to use multiple Visibility values.

Also, I think we have a ConvertBack method implemented, which you can check if worth mentioning.